.rspec

```diff
@@ -1,2 +1,2 @@
---format documentation
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.